README.md

```diff
@@ -138,3 +138,4 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License <a name="license"></a>
 This project is [MIT](./LICENSE) licensed.
+c:\Users\debby\Desktop\ugbe\MICRO VERSE CODES\Awesome_App_Es6-development_branch\.github c:\Users\debby\Desktop\ugbe\MICRO VERSE CODES\Awesome_App_Es6-development_branch\modules c:\Users\debby\Desktop\ugbe\MICRO VERSE CODES\Awesome_App_Es6-development_branch\screenshots c:\Users\debby\Desktop\ugbe\MICRO VERSE CODES\Awesome_App_Es6-development_branch\.eslintrc.json c:\Users\debby\Desktop\ugbe\MICRO VERSE CODES\Awesome_App_Es6-development_branch\.gitignore c:\Users\debby\Desktop\ugbe\MICRO VERSE CODES\Awesome_App_Es6-development_branch\.hintrc c:\Users\debby\Desktop\ugbe\MICRO VERSE CODES\Awesome_App_Es6-development_branch\.stylelintrc.json c:\Users\debby\Desktop\ugbe\MICRO VERSE CODES\Awesome_App_Es6-development_branch\index.css c:\Users\debby\Desktop\ugbe\MICRO VERSE CODES\Awesome_App_Es6-development_branch\index.html c:\Users\debby\Desktop\ugbe\MICRO VERSE CODES\Awesome_App_Es6-development_branch\index.js c:\Users\debby\Desktop\ugbe\MICRO VERSE CODES\Awesome_App_Es6-development_branch\LICENSE c:\Users\debby\Desktop\ugbe\MICRO VERSE CODES\Awesome_App_Es6-development_branch\package-lock.json c:\Users\debby\Desktop\ugbe\MICRO VERSE CODES\Awesome_App_Es6-development_branch\package.json
```

Comment:
- [ ] [Optioanl] Well, I would recommend deleting the unnecessary details you've added in the README file. It doesn't contribute positively to the overall appearance of the document. Keeping the README clean and concise is essential for clarity and easy understanding.
![image](https://github.com/EngruugbeJimmy/Awesome-App_New/assets/55840999/323b0f26-e6a2-483a-bfeb-003d967fd547)